.github/workflows/atlas.yml

```diff
@@ -25,24 +25,30 @@ jobs:
 
     - name: "installing geOrchestra root pom"
       run: ./mvnw install --non-recursive
+
     - name: installing georchestra-commons into .m2
-      run: ./mvnw install -pl commons
+      run: ./mvnw install -P-all,atlas -pl commons
+
     - name: install & check formating
       working-directory: atlas/
       run: ../mvnw --no-transfer-progress -B -Dfmt.action=validate install -Dadditionalparam=-Xdoclint:none -DskipTests
-    - name: run test
+
```

Comment:
Don't know why, but my comment has been lost :-(

I think you can refactor this into only one action, as using `-P-all,atlas` will take advantage of maven dependency resolution process, that should figure out by itself that it has to compile commons first:

```
[get rid of root pom & georchestra-commons]

    - name: install & check formating
      run: ./mvnw --no-transfer-progress -B -P-all,atlas -Dfmt.action=validate install -Dadditionalparam=-Xdoclint:none -DskipTests
    
   - name: run test
[...]
```